spec/chewy/rspec/update_index_spec.rb

```diff
@@ -113,7 +113,10 @@
     end
     specify do
       expect do
-        expect { DummiesIndex::Dummy.bulk body: [{index: {_id: 42, data: {}}}, {index: {_id: 43, data: {}}}, {delete: {_id: 41}}] }
+        expect do
+          DummiesIndex::Dummy.bulk body: [{index: {_id: 42, data: {}}}, {index: {_id: 43, data: {}}},
+                                          {delete: {_id: 41}}]
+        end
```

Comment:
Merging `end` with the next line `.to ...` would look better IMHO. But let's leave this for further specs refactoring.